packages/types/App.d.ts

```diff
@@ -1,6 +1,7 @@
 import type { Prisma } from "@prisma/client";
 
 import type { Tag } from "@calcom/app-store/types";
+import type { AppCategories } from "@calcom/prisma/enums";
```

Comment:
The AppCategories enum should be our single source of truth for all app categories available. The `variant` property didn't follow this enum.